src/openzaak/components/zaken/tests/test_zaken.py

```diff
@@ -943,6 +943,91 @@ def test_betalingsindicatie_weergave(self):
             zaak6["betalingsindicatieWeergave"], BetalingsIndicatie.nvt.label
         )
 
+    def test_zaak_autofill_einddatums(self):
+        url = reverse("zaak-list")
+        startdatum = date(2020, 1, 1)
+        self.zaaktype.servicenorm_behandeling = relativedelta(days=30)
+        self.zaaktype.save()
+
+        response = self.client.post(
+            url,
+            {
+                "zaaktype": f"http://testserver{self.zaaktype_url}",
+                "bronorganisatie": "517439943",
+                "verantwoordelijkeOrganisatie": "517439943",
+                "registratiedatum": str(startdatum),
+                "startdatum": str(startdatum),
+            },
+            **ZAAK_WRITE_KWARGS,
+        )
+
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED, response.data)
+        zaak = Zaak.objects.get(uuid=response.data["uuid"])
+
+        self.assertEqual(
+            zaak.einddatum_gepland,
+            startdatum + relativedelta(days=30),
+        )
+        self.assertEqual(
+            zaak.uiterlijke_einddatum_afdoening,
+            startdatum + relativedelta(days=30),
+        )
+
+    def test_zaak_met_eindatums(self):
+        url = reverse("zaak-list")
+        startdatum = date(2020, 1, 1)
+        einddatum_gepland = startdatum + relativedelta(days=10)
```

Comment:
Could you also make sure the servicenorm on the zaaktype is set here? That way we know it's not just being skipped because the servicenorm is not defined, but also because a date was passed explicitly